client/src/components/layout/TripTile.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const TripTile = (props) => {
+  let location = ""
+  if (props.trip.city) {
+    location = `${props.trip.city}, ${props.trip.country}`
+  } else {
+    location = `${props.trip.country}`
+  }
+
+  let numOfDays = ""
+  if (props.trip.numberOfDays) {
+    numOfDays = `Number Of Days: ${props.trip.numberOfDays}`
+  }
+  
+  return (
+    <div>
+      <h2>
+        <Link to={`/trips/${props.trip.id}`}>{props.trip.title}</Link>
```

Comment:
This Link would technically be broken since the Show page hasn't been built and there is no route in the React BrowserRouter that can handle this path - this should be added when working on the Show page feature